examples/kubernetes/quickstart.yaml

```diff
@@ -59,6 +59,15 @@ spec:
           name: backend
           ports:
             - containerPort: 3000
+          env:
```

Comment:
can you share the impact - before/after ?

Reply:
nvm just saw this https://github.com/envoyproxy/gateway/issues/591

Reply:
can you please attach a log :)

Reply:
before:
<img width="701" alt="image" src="https://user-images.githubusercontent.com/48784001/196476598-ee579e3f-ba69-46f7-a4a2-7f42d9f65ee3.png">

after:

<img width="702" alt="image" src="https://user-images.githubusercontent.com/48784001/196476293-f6bacfa2-f211-443c-961c-ecc929141123.png">